app/assets/javascripts/views/story_view.js

```diff
@@ -377,7 +377,6 @@ module.exports = FormView.extend({
 
           const $storyState = $('<div class="form-group" data-story-state></div>');
           $(div).append($storyState);
-
```

Comment:
Probably, this was a trailing space that was removed when I saved this file with editorconfig on my editor.